src/api/todos.ts

```diff
@@ -0,0 +1,36 @@
+import { Todo } from '../types/Todo';
+import { client } from '../utils/fetchClient';
+
+export const getTodos = (userId: number) => {
+  return client.get<Todo[]>(`/todos?userId=${userId}`);
+};
+
+export const sendNewTodo = async (title: string, userId: number) => {
```

Comment:
It's a good practice to add a return type for the functions. Consider adding a return type `Promise<Todo>` for `sendNewTodo` function. 🧐